packages/local_auth/lib/auth_strings.dart

```diff
@@ -2,6 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// This is a temporary ignore to allow us to land a new set of linter rules in a
+// series of manageable patches instead of one gigantic PR. It disables some of
+// the new lints that are already failing on this plugin, for this plugin. It
+// should be deleted and the failing lints addressed as soon as possible.
+// ignore_for_file: public_member_api_docs
```

Comment:
Originally I tried putting in an `analysis_options.yaml` for this plugin like everywhere else, but for some reason this particular plugin was having issues importing the `pedantic` package transitively when running `pub global tuneup`. No idea why as the package was showing as included in the `pubspec.lock` and nothing looked different from any of the other packages. Seems like some kind of bug to me in pub or the linter to me. I'm just working around it by putting the ignores in the source code directly instead. It's not ideal, but it shouldn't matter since in the long term this should be fixed anyway.